lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_merge_strings_files.rb

```diff
@@ -2,10 +2,14 @@ module Fastlane
   module Actions
     class IosMergeStringsFilesAction < Action
       def self.run(params)
-        UI.message "Merging strings files: #{params[:paths].inspect}"
+        destination = params[:destination]
+        # Include the destination as the first of the files (without key prefixes) to be included in the merged result
+        all_paths_list = { destination => nil }.merge(params[:paths_to_merge])
 
-        destination = params[:destination] || params[:paths].keys.first # Use first path if destination not provided, to do an in-place merge.
-        duplicates = Fastlane::Helper::Ios::L10nHelper.merge_strings(paths: params[:paths], output_path: destination)
+        UI.message "Merging strings files #{all_paths_list.inspect}"
+
+        File.write(destination, '') unless File.exist?(destination) # Create empty destination file if it does not exist yet
```

Comment:
Micro readability nitpick. How do you feel about moving this logic into its own, well named, method? 

Something like:

```diff
- File.write(destination, '') unless File.exist?(destination) # Create empty destination file if it does not exist yet
+ File.create_empty_if_not_existing(destination)
```

Reply:
I'm not sure it's worth it tbh, especially given it's only used in one place here (and makes for a long and potentially debatable name for an extension method 😛